### PR TITLE
recipe for benten 2019.09.13

### PR DESCRIPTION
**Pull request by pvanheus**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by pvanheus** [comment]
build will fail till [dukpy](https://github.com/conda-forge/staged-recipes/pull/9994) is merged.

File: recipes/benten/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/rabix/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e2fab99296ba80007d15754629483b72df5589c63a34d8b9af5a4873d938a8d7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```
Comment (xhochy) [inline recipes/benten/meta.yaml]: Looks like a pure Python package, please make this `noarch`
```suggestion
  noarch: python
```

File: recipes/benten/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  entry_points:
+    - benten-ls=benten.__main__:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - dukpy
```
Comment (xhochy) [inline recipes/benten/meta.yaml]: Are these two neeeded for building the package?

**Comment by pvanheus** [comment]
@xhochy thanks for the review. Looks like I'll need to move this to `bioconda` because it depends on `sevenbridges-python` which is in that channel. This is somewhat unfortunately because the CWL language that `benten` supports is not restricted to users in bioinformatics.

The only alternative I can see is moving `sevenbridges-python` to conda-forge.

**Comment by xhochy** [comment]
> The only alternative I can see is moving `sevenbridges-python` to conda-forge.

This is the preferred way actually.

**Comment by pvanheus** [comment]
The author removed the `sevenbridges-python` requirement in a new release, so upgraded recipe to that one.

**Comment by pvanheus** [comment]
@xhochy I've removed the `noarch: python` because that makes the build fail with an error like: `ImportError: cannot import name '_dukpy' from partially initialized module 'dukpy' (most likely due to a circular import) (/home/pvh/anaconda3/conda-bld/benten_1572889846352/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.8/site-packages/dukpy/__init__.py`

**Comment by beckermr** [comment]
```
conda.exceptions.ResolvePackageNotFound: 
  - boolean.py[version='>=3.6,<4.0.0']
```

LMK when you make further progress!

**Comment by pvanheus** [comment]
Thanks @beckermr  looks like kicking the CI fixed things!

**Pull request merged**